docs/source/cuda/device-management.rst

```diff
@@ -34,6 +34,7 @@ Users can then create a new context with another device.
 
 
 .. function:: numba.cuda.select_device(device_id)
+   :noindex:
```

Comment:
I don't think it does - my plan is to next rewrite the "Numba for CUDA GPUs" section so that it provides an explanation of how to use CUDA Python, referring forward to the reference section as necessary.